views/layout.html

```diff
@@ -29,6 +29,13 @@
     {% endif %}
   </head>
   <body style="margin: 0" data-make-url="{{ makeUrl }}" data-make-endpoint="{{ MAKE_ENDPOINT }}">
+    <div class="new-thimble-notice" id="new-thimble-banner">
+      <a href="#" title="Dismiss this notice" class="dismiss-notice"></a>
```

Comment:
if this is not a link, don't use an `<a>`. Use a `<button>` and style it to look however you need it to look. `<a href="#">` means "scroll to the top of the page"